alpha/pom.xml

```diff
@@ -35,6 +35,27 @@
     <module>alpha-server</module>
   </modules>
 
+  <dependencies>
+    <dependency>
+      <groupId>io.kamon</groupId>
+      <artifactId>kamon-core_2.12</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.kamon</groupId>
+      <artifactId>kamon-annotation_2.12</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.servicecomb.saga</groupId>
```

Comment:
alpha doesn't use saga-core , we should not add this dependency.

Reply:
ok.done